// Subset of http://google3/devtools/kokoro/api/proto/kokoro_api.proto
//
// Run these commands to regenerate kokoro_api_pb2.py:
//
//     apt-get install protobuf-compiler
//     cd ndk/scripts
//     protoc kokoro_api.proto --python_out .
//

syntax = "proto2";

package KokoroApi;

message GitOnBorgScmRevision {
  optional string name = 3;
  optional string sha1 = 1;
}

message MultiScmRevision {
  repeated GitOnBorgScmRevision git_on_borg_scm_revision = 2;
}

message BuildResult {
  optional MultiScmRevision multi_scm_revision = 12;
  repeated string build_artifacts = 2;
  map<string, string> env_vars = 14;
}

message BuildStatusResponse {
  optional string build_id = 5;
  optional BuildResult build_result = 2;
}
